pkg/reconciler/revision/resources/imagecache.go

```diff
@@ -27,15 +27,20 @@ import (
 )
 
 // MakeImageCache makes an caching.Image resources from a revision.
-func MakeImageCache(rev *v1.Revision) *caching.Image {
-	image := rev.Status.DeprecatedImageDigest
+func MakeImageCache(rev *v1.Revision, name string) *caching.Image {
```

Comment:
Shall we pass in `containerName, imageDigest string` to not need to iterate twice?

Reply:
Currently passing only containerName not the digest
because [here](https://github.com/knative/serving/pull/7661/files#diff-83d81ad460499e87cc0d935f33c5228eR120) I don't get the digest value
when i do **container.Image**

Reply:
If you iterate over status, you'd get both, right?

Reply:
Ah got it modified.
Please take a look :)